cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailures.java

```diff
@@ -93,7 +93,8 @@ public void configure(Map<String, ?> configs) {
                                                            allowCapacityEstimation,
                                                            excludeRecentlyDemotedBrokers,
                                                            excludeRecentlyRemovedBrokers,
-                                                           _anomalyId)
+                                                           _anomalyId,
+                                                           String.format("Self healing for broker failure : %s", this))
```

Comment:
Nit (applies to similar uses in other anomaly types for the sake of consistency): `broker failure` -> `%s` that uses `AnomalyType.BROKER_FAILURE`?